ballerina/build.gradle

```diff
@@ -203,6 +214,20 @@ task copyOtherRepos(type: Copy) {
             from "${artifactExtractedPath}/caches/jar"
         }
     }
+
+    /* C2C Libraries */
+    configurations.ballerinaC2cLibs.resolvedConfiguration.resolvedArtifacts.each { artifact ->
+        def artifactExtractedPath = "${buildDir}/target/extracted-distributions/" + artifact.name + "-zip"
+        into("bre/lib/") {
+            from "${artifactExtractedPath}/libs"
+        }
+        into("docs/") {
+            from "${artifactExtractedPath}/docs"
+        }
+        into("bir-cache/") {
+            from "${artifactExtractedPath}/caches/bir"
+        }
```

Comment:
Shouldn't we need to pack these C2C library artifacts to the new packerina cache?.
```gradle
// pack to new cache
into("cache/bir/") {
    from "${artifactExtractedPath}/caches/bir"
}
into("cache/balo/") {
    from "${artifactExtractedPath}/caches/balo"
}
into("cache/jar/") {
    from "${artifactExtractedPath}/caches/jar"
}
```

@anuruddhal @pramodya1994 @hevayo

Reply:
No. 
The annotation processors work by copying `bir-cache` and `libs` only.  
Docker and k8s annotation processors do the same.

Reply:
Yes, but AFAIK the new packerina implementation is going to get rid of existing caches and move on to the new cache. I just want to confirm it.

Reply:
@BuddhiWathsala You are right as per our plan we have to pack it there. But with the new package API change we need to revisit this so lets merge it.